### ISSUE TITLE
Animation bug

### ISSUE THREAD

**Issue by sekwah41** [sekwah41/Naruto-Mod#5]
Animation updates fire half as many times as the render event tick. Find out what is causing this as it seems exactly half.

Note: Check the fps settings just to make sure as it could be that the render event calls whenever possible and not actually when the world renders.

If its not fps then just create a second delta tracker for entity update events by adding up all the values of delta and resetting it at the end of the render.

Basically large visual problem to others, its just irritating. Also make sure it doesnt have the jitter when changing anims like old versions :)

**Comment by sekwah41** [comment]
Fixed and will be in next patch or update
